qb_waypoint_manager/qb_waypoint waypoint
---
bool success